src/components/FeedInfo.tsx

```diff
@@ -37,14 +38,15 @@ interface EditFeedState {
 export interface DispatchProps {
     onAddFeed: (feed: Feed) => void;
     onRemoveFeed: (feed: Feed) => void;
+    onDownloadFeed: (feed: Feed) => void;
```

Comment:
This is not necessary, because the onAddFeed implementation in the container also triggers the download.